MinecraftClient/Settings.cs

```diff
@@ -277,6 +278,9 @@ public static void LoadSettings(string settingsfile)
                                                 case "type": AccountType = argValue == "mojang" 
                                                         ? ProtocolHandler.AccountType.Mojang 
                                                         : ProtocolHandler.AccountType.Microsoft; break;
+                                                case "method": LoginMethod = argValue.ToLower() == "mcc"
+                                                        ? "mcc"
+                                                        : "browser"; break;
```

Comment:
I think the default value on invalid input should be `mcc` since it's the default value above in `LoginMethod`.